api/v1/recipe_groups/views.py

```diff
@@ -18,11 +17,19 @@ class CuisineViewSet(viewsets.ModelViewSet):
 
     Uses `title` as the PK for any lookups.
     """
-    queryset = Cuisine.objects.all()
-    serializer_class = CuisineSerializer
+    serializer_class = serializers.CuisineSerializer
     permission_classes = (permissions.IsAuthenticatedOrReadOnly,
                           IsOwnerOrReadOnly)
-    lookup_field = 'title'
+    lookup_field = 'slug'
+
+    def get_queryset(self):
+        query = Cuisine.objects
+
+        if 'course' in self.request.query_params:
+            course = Course.objects.get(slug=self.request.query_params.get('course'))
+            query = query.filter(recipe__course=course)
+
+        return query.annotate(total=Count('recipe', distinct=True))
```

Comment:
`annotate` and `Count` is what does the group by and count in the ORM, can't really move it elsewhere.